frontend/model/contracts/group.js

```diff
@@ -1286,8 +1361,11 @@ sbp('chelonia/defineContract', {
       process ({ meta }, { state, getters }) {
         const period = getters.periodStampGivenDate(meta.createdDate)
         const current = getters.groupSettings?.distributionDate
-
-        if (current !== period) {
+        const inWaitingPeriod = !current || new Date().toISOString() < current
```

Comment:
Another `new Date()`!

Reply:
Hmm I'm pretty sure I had fixed this one, using `meta.createdDate` instead. Seems like I somehow overwrote my own changes using git incorrectly 😭